vendor/bat-native-ledger/include/bat/ledger/global_constants.h

```diff
@@ -20,6 +20,7 @@ const char kWalletAnonymous[] = "anonymous";
 const char kWalletUphold[] = "uphold";
 const char kWalletUnBlinded[] = "blinded";
 const char kWalletBitflyer[] = "bitflyer";
+const char kWalletGemini[] = "gemini";
```

Comment:
these consts should go away and use the enum https://github.com/brave/brave-core/pull/8566/files#diff-5b5d5e1ed1b6101b793f102186a3734c64221d5759013a034abed5ad3d83e0b2R23 instead

Reply:
These constants are used extensively in the rewards code base, removing these would be a really big task. I filed a follow-up issue to remove these constants and use an enum instead - https://github.com/brave/brave-browser/issues/16815